cfme/services/catalogs/catalog_items/catalog_bundles.py

```diff
@@ -103,7 +103,7 @@ def create(self, name, catalog_items=None, catalog=None, description=None, displ
                 "StateMachines", "ServiceProvision_Template", "CatalogItemInitialization")
             view.apply_button.click()
         for cat_item in catalog_items:
-            view.resources.fill({'select_resource': cat_item})
+            view.resources.fill({'select_resource': cat_item.name})
```

Comment:
Required :
This will break the services bundle tests .
You have to pass the name from your tests 
Ex : https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/services/test_generic_service_catalogs.py#L82